app/build.gradle

```diff
@@ -84,6 +84,11 @@ def getLastFMApiKey() {
     return value != null ? value : ''
 }
 
+ext {
+    exoplayerVersion = '2.11.4'
```

Comment:
I don't see this version string being connected to bot's yml.

Reply:
The dependabot gradle plugin does this automatically, in this case.

What is still not possible however, is something like:
```
ignore:
    - dependency-name: "iconicsVersion"
    - versions: ["5.+"]
```

Reply:
In this case dependabot does this automatically. However it cannot use variables as labels for ignoring version ranges.

Reply:
I see, thanks